crypto/rsa/rsa_oaep.c

```diff
@@ -129,6 +121,15 @@ int ossl_rsa_padding_add_PKCS1_OAEP_mgf1_ex(unsigned char *to, int tlen,
     return rv;
 }
 
+int RSA_padding_add_PKCS1_OAEP(unsigned char *to, int tlen,
+                               const unsigned char *from, int flen,
+                               const unsigned char *param, int plen)
+{
+    return ossl_rsa_padding_add_PKCS1_OAEP_mgf1_ex(to, tlen, from, flen,
+                                                   param, plen, NULL, NULL);
+}
+
+
```

Comment:
Why was this moved?

Reply:
To remove a warning of missing function definition. Previously this function was used before definition. Moving it to before it is used removes that warning.

Reply:
I *hate* warnings in code :-). They hide other errors (and indeed whilst I was tidying up warnings I found some genuine bugs that I fixed).